packages/app/src/app/pages/NewDashboard/Components/Repo/ImportRepo.tsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import { useOvermind } from 'app/overmind';
+import { Stack, Text, Button, Icon } from '@codesandbox/components';
+import css from '@styled-system/css';
+
+export const ImportRepo = () => {
+  const { actions } = useOvermind();
+
+  const onClick = () => actions.modalOpened({ modal: 'newSandbox' });
```

Comment:
There's a new one, I think it's 
```suggestion
  const onClick = () => actions.modals.newSandboxModal.open({});
```

Reply:
updated